pkg/zonemgr/zone_manager.go

```diff
@@ -28,52 +28,26 @@ func NewZoneManager() *ZoneManager {
 }
 
 func (zoneMgr *ZoneManager) init() {
+	domain := os.Getenv("DOMAIN")
+	nameServerIP := os.Getenv("NAME_SERVER_IP")
```

Comment:
better to have consts for the env var names at the beginning of the file please

Reply:
done